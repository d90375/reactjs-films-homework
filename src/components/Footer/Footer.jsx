import React from "react";
import "./Footer.scss";

const Footer = () => {
  return (
    <div className="footer">
      <div className="footer__title">
        <span>FILMS</span>
      </div>
      <div className="footer__copyright">
        <span>Copyright Â© 2018 d90375</span>
      </div>
    </div>
  );
};

export default Footer;
