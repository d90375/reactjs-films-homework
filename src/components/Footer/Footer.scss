.footer {
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 100%;
  height: 263px;
  box-shadow: 0 10px 40px rgba(0, 0, 0, 0.04);
  background-color: #fff;
}

.footer__title {
  margin-top: 75px;
  color: #0aaee4;
  font-family: "Helvetica Neue", serif;
  font-size: 50px;
  font-weight: 400;
  text-transform: uppercase;
  letter-spacing: 8px;
}

.footer__copyright {
  margin-top: 75px;
  opacity: 0.4;
  color: #242b2d;
  font-family: "Helvetica Neue", serif;
  font-size: 24px;
  font-weight: 400;
  letter-spacing: 1.2px;
}
