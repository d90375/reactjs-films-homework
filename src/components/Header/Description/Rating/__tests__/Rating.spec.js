import React from "react";
import Footer from "../Rating";

describe("Rating", () => {
  let component;

  it("should render Rating component without props", () => {
    component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });

  it("should render Rating component with correct props", () => {
    component = shallow(<Footer voteAverage={9} />);
    expect(component).toMatchSnapshot();
  });

  it("should render Rating component with not correct props", () => {
    component = shallow(<Footer voteAverage={99} />);
    expect(component).toMatchSnapshot();
  });
});
