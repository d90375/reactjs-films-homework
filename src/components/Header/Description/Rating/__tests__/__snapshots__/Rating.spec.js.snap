// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating should render Rating component with correct props 1`] = `
<div
  className="rating"
>
  <div
    className="star"
    key="0"
  >
    ★
  </div>
  <div
    className="star"
    key="1"
  >
    ★
  </div>
  <div
    className="star"
    key="2"
  >
    ★
  </div>
  <div
    className="star"
    key="3"
  >
    ★
  </div>
  <div
    className="star"
    key="4"
  >
    ★
  </div>
  <div
    className="container"
  >
    <span
      className="number"
    >
      4.5
    </span>
  </div>
</div>
`;

exports[`Rating should render Rating component with not correct props 1`] = `
<div
  className="rating"
>
  <div
    className="container"
  >
    <span
      className="number"
    >
      0.0
    </span>
  </div>
</div>
`;

exports[`Rating should render Rating component without props 1`] = `
<div
  className="rating"
>
  <div
    className="container"
  >
    <span
      className="number"
    >
      0.0
    </span>
  </div>
</div>
`;
